cmd/create.go

```diff
@@ -153,34 +153,7 @@ to quickly create a Cobra application.`,
 			}
 			time.Sleep(45 * time.Second)
 		}
-
-		informUser("Syncing the registry application")
```

Comment:
I am not 100% sure we can remove this.. 
In some installs, we got in a state that this synch was needed, due to some softserve/argo miscommunication.

I believe we should not block, the installer in case is already happening, but not 100% on remove it to ensure consistency.

Reply:
I had 2 successful runs for different regions using this PR.